openapi.yml

```
@@ -5523,6 +5683,255 @@ components:
+                invite:
+                    id: 39
+                    first_name: some text
+                    last_name: some text
+                    start_date: '2018-02-10T09:30Z'
+                    location:
+                        id: 21
+                        name: some text
+                    watchlist_colour: GREEN
+                    hosts:
+                        -
+                            id: 16
+                            email: some text
+                            first_name: some text
+                            last_name: some text
+                            profile_pic_url: some text
+                            department: some text
+                            mobile_number: some text
+                        -
+                            id: 92
+                            email: some text
+                            first_name: some text
+                            last_name: some text
+                            profile_pic_url: some text
+                            department: some text
+                            mobile_number: some text
+                    invite_watchlist:
+                        id: 91
+                        external_colours:
+                            - some text
+                            - some text
+                        internal_colours:
+                            - some text
+                            - some text
+                    end_date: '2018-02-10T09:30Z'
+                    email: some text
+                    mobile_number: some text
+                visitor:
+                    id: 20
+                    active: true
+                    company: some text
+                    created_via: some text
+                    email: some text
+                    first_name: some text
+                    last_name: some text
+                    mobile: some text
+                    note: some text
+                    profile_pic_content_type: some text
+                    profile_pic_file_name: some text
+                    profile_pic_file_size: some text
+                    profile_pic_updated_at: some text
+                    watchlist_level: some text
+                    created_at: '2018-02-10T09:30Z'
+                    updated_at: '2018-02-10T09:30Z'
+        GuestResponse:
+            title: Root Type for GuestResponse
+            description: The data collected from the response on a Registration
+            type: object
+            properties:
+                title:
+                    description: Page title
+                    type: string
+                sequence:
+                    format: int32
+                    type: integer
+                id:
+                    format: int32
+                    description: Unique identifier
+                    type: integer
+                page_type:
+                    description: Enum
+                    type: string
```

If this is an `enum`, the valid values can be defined this way:

```suggestion
                    type: string
                    enum:
                             - ValidTypeOne
                             - ValidTypeTwo

```